uvm/create.go

```diff
@@ -205,7 +206,12 @@ func Create(opts *UVMOptions) (*UtilityVM, error) {
 		}
 	}
 
-	hcsSystem, err := hcs.CreateComputeSystem(uvm.id, hcsDocument, opts.AdditionHCSDocumentJSON)
+	fullDoc, err := mergemaps.MergeJSON(hcsDocument, ([]byte)(opts.AdditionHCSDocumentJSON))
```

Comment:
This becomes inefficient marshalling/unmarshalling needlessly when in the 99% case, this will be `""`